finmanflask/auth.py

```diff
@@ -0,0 +1,62 @@
+from flask import (
+    Blueprint, flash, g, redirect, render_template, request, session, url_for, jsonify, Response, json 
+)
+from finmanflask.db import get_engine
+from sqlalchemy.orm import Session
+from sqlalchemy.exc import IntegrityError
+from werkzeug.security import check_password_hash, generate_password_hash
+
+bp = Blueprint('auth', __name__, url_prefix='/auth')
+
+def validate(field, value):
+    """Validate the field of a given value"""
+    # TODO: implement
+    return None
+
+
+@bp.route('/register', methods=('POST', ))
+def register():
+
+    input_error = {"status" : 400, "error": "Bad Request", "message":"Invalid input data", "details":[]}
+    integrity_error_message = {"status": 409, "error": "Username is already taken."}
+    input_error_throw = False
+
+
+    for key in ['username', 'email', 'password']:
```

Comment:
Can't we just have it in one `for` loop for this? Then we could also avoid using `input_error_throw` and just return error message